samples/applets/create-document-group.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * to run create document group applet from the project root folder type in your console:
+ * > node samples/applets/create-document-group <cliend_id> <client_secret> <username> <password> <group_name> <signature_stringified> <filepath>
```

Comment:
lets implement the applet in the way to pass document ids and group name and see the result